src/main/java/dev/jbang/cli/Export.java

```diff
@@ -502,3 +498,234 @@ private Path getJlinkOutputPath() {
 		return outputPath;
 	}
 }
+
+abstract class BaseExportProject extends BaseExportCommand {
+
+	@CommandLine.Option(names = { "--group", "-g" }, description = "The group ID to use for the exported project.")
+	String group;
+
+	@CommandLine.Option(names = { "--artifact",
+			"-a" }, description = "The artifact ID to use for the exported project.")
+	String artifact;
+
+	@CommandLine.Option(names = { "--version", "-v" }, description = "The version to use for the exported project.")
+	String version;
+
+	@Override
+	int apply(BuildContext ctx) throws IOException {
+		Path projectDir = exportMixin.getOutputPath("");
+		if (projectDir.toFile().exists()) {
+			if (exportMixin.force) {
+				Util.deletePath(projectDir, false);
+			} else {
+				Util.errorMsg("Cannot export as " + projectDir + " already exists. Use --force to overwrite.");
+				return EXIT_INVALID_INPUT;
+			}
+		}
+		info("Exporting as " + getType() + " project to: " + projectDir);
+
+		Project prj = ctx.getProject();
+		if (prj.isJar() || prj.getMainSourceSet().getSources().isEmpty()) {
+			Util.errorMsg("You can only export source files");
+			return EXIT_INVALID_INPUT;
+		}
+
+		if (prj.getGav().isPresent()) {
+			MavenCoordinate coord = MavenCoordinate.fromString(prj.getGav().get()).withVersion();
+			if (group == null) {
+				group = coord.getGroupId();
+			}
+			if (artifact == null) {
+				artifact = coord.getArtifactId();
+			}
+			if (version == null) {
+				version = coord.getVersion();
+			}
+		}
+
+		if (group == null) {
+			group = "org.example.project";
+		}
+		artifact = artifact != null ? artifact
+				: Util.getBaseName(Objects.requireNonNull(prj.getResourceRef().getFile()).getFileName().toString());
+		version = version != null ? version : MavenCoordinate.DEFAULT_VERSION;
+
+		createProjectForExport(ctx, projectDir);
+
+		return EXIT_OK;
+	}
+
+	private void createProjectForExport(BuildContext ctx, Path projectDir) throws IOException {
+		Project prj = ctx.getProject();
+		String projectName = projectDir.getFileName().toString();
+		Path tmpProjectDir = Files.createTempDirectory(projectName);
```

Comment:
This line, plus the `Files.move()` on line 590 won't work on Windows if the directory pointed to by `projectDir` is on a different drive than `%TEMP%`.

So I'd just skip the temp folder and immediately copy/create any files directly into the target folder.

Reply:
Thanks! I will change it accordingly.